Created PMK brand folder including posey_split keyboards with default and via keymaps. (#23258)
qmk/qmk_firmware: keyboards/pmk/posey_split/posey_split_v4/readme.md

# posey_split_v4

> ![posey_split_v4](https://i.imgur.com/g3yf6al.png)
> 
> *A short description of the keyboard/project*
> 
> * Keyboard Maintainer: [Javier Oliver](https://github.com/joliverMI)
> * Hardware Supported: posey_split_v4 and logic_module_v1
> * Hardware Availability: poseysmechanicalkeyboards.com
> 
> Make example for this keyboard (after setting up your build environment):
> 
>     make posey_split_v4:default
> 
> Flashing example for this keyboard:
> 
>     make posey_split_v4:default:flash

Reviewer: ```suggestion
![posey_split_v4](https://i.imgur.com/g3yf6alh.png)

* Keyboard Maintainer: [Javier Oliver](https://github.com/joliverMI)
* Hardware Supported: posey_split_v4 and logic_module_v1
* Hardware Availability: poseysmechanicalkeyboards.com

Make example for this keyboard (after setting up your build environment):

    make pmk/posey_split/posey_split_v4:default
    
Flashing example for this keyboard:

    make pmk/posey_split/posey_split_v4:default:flash
```
